DSA_Training/BinaryTreeHeight.java

```diff
@@ -0,0 +1,109 @@
+import java.util.LinkedList;
+import java.util.Queue;
+import java.util.Scanner;
+
+class TreeNode {
```

Comment:
Please, add here proper comments related to this class

Reply:
done